src/test/java/org/sagebionetworks/web/unitclient/widget/table/modal/fileview/ScopeWidgetTest.java

```diff
@@ -213,73 +221,28 @@ public void testConfigureUnsupportedViewTypeMask() {
 
     // Simulate clicking the edit button
     widget.onEditScopeAndMask();
-
-    // The mask editor should not be visible
-    verify(mockView).setEditMaskVisible(false);
-  }
-
-  @Test
-  public void testConfigureHappyCaseProjectView() {
-    when(mockEntityView.getType()).thenReturn(ViewType.project);
-    // configure with an entityview, edit the scope, and save.
-    boolean isEditable = true;
-    widget.configure(mockBundle, isEditable);
-
-    // The edit button should be visible
-    verify(mockView).setEditMaskAndScopeButtonVisible(true);
-
-    // The view scope widget should be configured and not be editable
-    verify(mockViewScopeWidget)
-      .configure(mockReferencesWithoutVersions, false, TableType.project_view);
-
-    // Simulate clicking the edit button
-    widget.onEditScopeAndMask();
-
-    // The mask editor should not be visible
-    verify(mockView).setEditMaskVisible(false);
-  }
-
-  @Test
-  public void testConfigureFileView() {
-    when(mockEntityView.getType()).thenReturn(ViewType.file);
-    boolean isEditable = true;
-
-    widget.configure(mockBundle, isEditable);
-    widget.onEditScopeAndMask();
-
-    // The mask editor should be visible
-    verify(mockView).setEditMaskVisible(true);
-
-    // Show File View options for project view
-    verify(mockView).setIsFileSelected(true);
-    verify(mockView).setIsFolderSelected(false);
-    verify(mockView).setIsTableSelected(false);
-    verify(mockView).setIsDatasetSelected(false);
   }
 
   @Test
-  public void testConfigureFileAndTableView() {
+  public void testConfigureView() {
     when(mockEntityView.getType()).thenReturn(ViewType.file_and_table);
     boolean isEditable = true;
 
     widget.configure(mockBundle, isEditable);
     widget.onEditScopeAndMask();
 
-    // The mask editor should be visible
-    verify(mockView).setEditMaskVisible(true);
-
-    // Show File View options for project view
-    verify(mockView).setIsFileSelected(true);
-    verify(mockView).setIsFolderSelected(false);
-    verify(mockView).setIsTableSelected(true);
-    verify(mockView).setIsDatasetSelected(false);
-
-    // verify update view type from file+table to file
-    when(mockView.isFileSelected()).thenReturn(true);
-    widget.updateViewTypeMask();
-    widget.onSave();
-
-    verify(mockEntityView).setViewTypeMask(ViewTypeMask.File.getMask());
+    // verify new editor is configured
+    ArgumentCaptor<EntityViewScopeEditorModalProps.Callback> onUpdateArgumentCaptor =
+      ArgumentCaptor.forClass(EntityViewScopeEditorModalProps.Callback.class);
+    ArgumentCaptor<EntityViewScopeEditorModalProps.Callback> onCancelArgumentCaptor =
+      ArgumentCaptor.forClass(EntityViewScopeEditorModalProps.Callback.class);
+    verify(mockEntityViewScopeEditorModalWidget)
+      .configure(
+        eq(entityId),
+        onUpdateArgumentCaptor.capture(),
+        onCancelArgumentCaptor.capture(),
```

Comment:
Since you aren't using the captors, you can just do
```suggestion
        any(EntityViewScopeEditorModalProps.Callback.class),
        any(EntityViewScopeEditorModalProps.Callback.class),
```